sagemaker-autopilot/housing_prices/autopilot_california_housing.ipynb

```diff
@@ -0,0 +1,904 @@
+{
```

Comment:
<p>Grammar nit - <code>how the model perform</code> -&gt; <code>how the model performs</code> ﻿ </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3026/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>